public/locales/en/translation.json

```diff
@@ -56,7 +57,8 @@
     "importBackendFailed": "Importing from backend failed.",
     "cannotExportEmptyCanvas": "Cannot export empty canvas.",
     "couldNotCopyToClipboard": "Couldn't copy to clipboard. Try using Chrome browser.",
-    "copiedToClipboard": "Copied to clipboard: {{url}}"
+    "copiedToClipboard": "Copied to clipboard: {{url}}",
```

Comment:
You removed this one from codebase, so let's remove it from all translations. WTBS, this means we'll need to re-translate it for the new label below.